crypto_analytics/collection/data_source/base.py

```diff
@@ -18,3 +21,34 @@ def write(self, filepath: str):
     @abstractmethod
     def get_time(self):
         pass
+
+
+class OHLCDataSource(DataSource):
```

Comment:
I though renaming `FinancialDataSource` to `OHLCDataSource` would make sense because there are probably more types of financial data.